internal/infrastructure/ai/fake_adapter.go

```
@@ -0,0 +1,146 @@
+		ClaimIDs:   claimIDs,
+		SourceRefs: sourceRefs,
+		BriefText:  brief,
+	}
+	return out, nil
+}
+
+// inputDigest produces a stable, content-addressed hex digest of the input.
+// Used to derive synthetic IDs that are deterministic but distinct per input.
+func inputDigest(in port.WeeklyBriefInput) string {
+	h := sha256.New()
+	fmt.Fprintf(h, "%s\x00%s\x00%s\x00%s\x00%s",
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)